raiden/tests/utils/mock_client.py

```diff
@@ -315,8 +315,10 @@ def channelnew_filter(self):
 
 
 class NettingChannelMock(object):
-    def __init__(self, asset_address, peer1, peer2, settle_timeout,  # pylint: disable=too-many-arguments
+    def __init__(self, asset_address, peer1, peer2, settle_timeout,
```

Comment:
These tools consider *character* length, including comments, need to splite otherwise the linter fail: